project_4.html

```diff
@@ -0,0 +1,524 @@
+
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-schematics">Schematics</h2>
+			<div class="accordion mb-5" id="schemAccordian">
+				<div class="accordion-item">
+					<h2 class="accordion-header">
+						<button class="accordion-button" type="button" data-bs-toggle="collapse"
+							data-bs-target="#schemCollapse1" aria-expanded="true" aria-controls="schemCollapse1">
+							Schematic A (LED Blinker)
+						</button>
+					</h2>
+					<!--Pin to page?-->
+					<div id="schemCollapse1" class="accordion-collapse collapse show" data-bs-parent="#schemAccordian">
+						<div class="accordion-body text-center">
+							<img src='./assets/images/project_4/ops_schem_proj4a.png' class="pj-schem">
+						</div>
+					</div>
+				</div>
+
+				<div class="accordion-item">
+					<h2 class="accordion-header">
+						<button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
+							data-bs-target="#schemCollapse2" aria-expanded="true" aria-controls="schemCollapse2">
+							Schematic B (LED Dimmer)
+						</button>
+					</h2>
+					<!--Pin to page?-->
+					<div id="schemCollapse2" class="accordion-collapse collapse" data-bs-parent="#schemAccordian">
+						<div class="accordion-body text-center">
+							<img src='./assets/images/project_4/ops_project_4_schematic_b.png' class="pj-schem">
+						</div>
+					</div>
+				</div>
+
+				<div class="accordion-item">
+					<h2 class="accordion-header">
+						<button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
+							data-bs-target="#schemCollapse3" aria-expanded="true" aria-controls="schemCollapse3">
+							Schematic C (RGB LED Dimmer)
+						</button>
+					</h2>
+					<!--Pin to page?-->
+					<div id="schemCollapse3" class="accordion-collapse collapse" data-bs-parent="#schemAccordian">
+						<div class="accordion-body text-center">
+							<img src='./assets/images/project_4/ops_schem_proj4c.png' class="pj-schem">
+						</div>
+					</div>
+				</div>
+				<div class = "mb-3">
+				</div>
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-instructions">Instructions</h2>
+			<h3 class="mb-3 fw-light" id="pj-heading-checkpoint1">Checkpoint 1</h3>
+			<ol class="mb-5 list-group list-group-numbered">
+				<li class="list-group-item">
+					Build the circuit from <strong>Schematic A</strong> (LED Blinker) on your breadboard. You will design a blinking LED using the Arduino Nano.
+					<div class="alert alert-info mt-2" role="alert">
+						Use the pseudo code below to program an LED that blinks on and off.
+					</div>
+					<div class="alert alert-info mt-2" role="alert">
+						These functions may be helpful: <code class="pj-code language-arduino">Serial.begin()</code>, <code class="pj-code language-arduino">pinMode()</code>, <code class="pj-code language-arduino">digitalWrite()</code>, <code class="pj-code language-arduino">delay()</code>
+					</div>
+					<pre class="mt-3"><code class="pj-code language-plaintext">// Assign variable to pin number for LED
```

Comment:
The psuedo code is not color highlighted correctly. Please make sure this class is set to language-arduino. The same applies to all other code snippets.